### PR TITLE
Add blog post about new releases

### PR DESCRIPTION
**Pull request by dr0i**
Titled: "Metafacture releases: Core 6.0.0, Fix 1.0.0, Playground 1.0.0".

See https://github.com/metafacture/metafacture-core/issues/515 .

We really should more promote the releases of the other `metafacture` family members. This is a try to put this together in one place (and honestly, it would be unnecessarily complex to blog post about all 3 of them in a post on it's own.)

The title was proposed by @blackwinter .

Feel free to change everything or anything :)

### CODE REVIEW

File: content/blog/metafacture-releases_core-6.0.0_fix-1.0.0_playground-1.0.0/index.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+date: "2024-03-12T17:05:03.2300Z"
+description: "Changes coming with the releases of metafacture-core 6.0.0,
+metafacture-fix 1.0.0 and metafacture-playground 1.0.0"
+authors: [{lastname: "Christoph",
+           firstname: "Pascal"}]
+---
+
+*Preamble*
+
+Most remarkable is certainly the switch to `Java 11`, hence the new Major
```
Comment (blackwinter) [inline content/blog/metafacture-releases_core-6.0.0_fix-1.0.0_playground-1.0.0/index.md]:
```suggestion
Most remarkable is certainly the switch to `Java 11`, hence the new major
```

File: content/blog/metafacture-releases_core-6.0.0_fix-1.0.0_playground-1.0.0/index.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+versions.
+
+This post describes the new developments coming in:
+
+a) [`metafacture-core release
+    6.0.0`](https://central.sonatype.com/namespace/org.metafacture) since the [release of
+ metafacture-core-5.7.0](https://blog.metafacture.org/metafacture-5.7.0/) in November 2023.
```
Comment (blackwinter) [inline content/blog/metafacture-releases_core-6.0.0_fix-1.0.0_playground-1.0.0/index.md]:
```suggestion
 metafacture-core 5.7.0](https://blog.metafacture.org/metafacture-5.7.0/) in November 2023.
```

File: content/blog/metafacture-releases_core-6.0.0_fix-1.0.0_playground-1.0.0/index.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+
+b) [`metafacture-fix-1.0.0`](https://github.com/metafacture/metafacture-fix/releases/tag/1.0.0).
```
Comment (blackwinter) [inline content/blog/metafacture-releases_core-6.0.0_fix-1.0.0_playground-1.0.0/index.md]:
```suggestion
b) [`metafacture-fix 1.0.0`](https://github.com/metafacture/metafacture-fix/releases/tag/1.0.0).
```

File: content/blog/metafacture-releases_core-6.0.0_fix-1.0.0_playground-1.0.0/index.md [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+    (metafacture-fix is ["an alternative to configuring data transformations with
+    Metamorph. Inspired by Catmandu Fix"](https://github.com/metafacture/metafacture-fix)).
```
Comment (blackwinter) [inline content/blog/metafacture-releases_core-6.0.0_fix-1.0.0_playground-1.0.0/index.md]:
```suggestion
    Metamorph, inspired by Catmandu Fix"](https://github.com/metafacture/metafacture-fix)).
```

File: content/blog/metafacture-releases_core-6.0.0_fix-1.0.0_playground-1.0.0/index.md [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+
+c)  `metafacture-playground-1.0.0`.
```
Comment (blackwinter) [inline content/blog/metafacture-releases_core-6.0.0_fix-1.0.0_playground-1.0.0/index.md]:
```suggestion
c) `metafacture-playground 1.0.0`.
```

File: content/blog/metafacture-releases_core-6.0.0_fix-1.0.0_playground-1.0.0/index.md [hunk 6]
```diff
@@ -0,0 +1,80 @@
...
+    (metafacture-playground is ["a web application to play around with Metafactures languages Fix, Flux and Morph inspired by the JSON-LD Playground"](https://github.com/metafacture/metafacture-playground/)).
```
Comment (blackwinter) [inline content/blog/metafacture-releases_core-6.0.0_fix-1.0.0_playground-1.0.0/index.md]:
```suggestion
    (metafacture-playground is ["a web application to play around with Metafacture's languages Fix, Flux and Morph, inspired by the JSON-LD Playground"](https://github.com/metafacture/metafacture-playground/)).
```

File: content/blog/metafacture-releases_core-6.0.0_fix-1.0.0_playground-1.0.0/index.md [hunk 7]
```diff
@@ -0,0 +1,80 @@
...
+# this will be replaced by the toc
+```
+
+## metafacture-core
+
+### New Features
+- IO: Open gzip compressed content from HTTP [#511](https://github.com/metafacture/metafacture-core/issues/511)
+
+### Bug fixes
+- IO:  Fix misconception of "Content-Encoding" [#513](https://github.com/metafacture/metafacture-core/issues/513)
```
Comment (blackwinter) [inline content/blog/metafacture-releases_core-6.0.0_fix-1.0.0_playground-1.0.0/index.md]:
```suggestion
- IO: Fix misconception of "Content-Encoding" [#513](https://github.com/metafacture/metafacture-core/issues/513)
```

File: content/blog/metafacture-releases_core-6.0.0_fix-1.0.0_playground-1.0.0/index.md [hunk 8]
```diff
@@ -0,0 +1,80 @@
...
+- IO: Close resources [#514](https://github.com/metafacture/metafacture-core/issues/514)
+
+### Others
+- Provide examples for every flux-command [#488](https://github.com/metafacture/metafacture-core/issues/488)
+- Upgrade Gradle wrapper [#520](https://github.com/metafacture/metafacture-core/issues/520)
+- Switch to Java version 11 [#518](https://github.com/metafacture/metafacture-core/issues518)
```
Comment (blackwinter) [inline content/blog/metafacture-releases_core-6.0.0_fix-1.0.0_playground-1.0.0/index.md]:
```suggestion
- Switch to Java version 11 [#518](https://github.com/metafacture/metafacture-core/issues/518)
```

File: content/blog/metafacture-releases_core-6.0.0_fix-1.0.0_playground-1.0.0/index.md [hunk 9]
```diff
@@ -0,0 +1,80 @@
...
+
+## metafacture-fix
+
+### New Features
+
+- Add extension snippets for all do binds [#302](https://github.com/metafacture/metafacture-fix/issues/302)
+- Upgrade Gradle wrapper [#283](https://github.com/metafacture/metafacture-fix/issues/302)
```
Comment (blackwinter) [inline content/blog/metafacture-releases_core-6.0.0_fix-1.0.0_playground-1.0.0/index.md]:
```suggestion
- Upgrade Gradle wrapper [#283](https://github.com/metafacture/metafacture-fix/issues/283)
```

File: content/blog/metafacture-releases_core-6.0.0_fix-1.0.0_playground-1.0.0/index.md [hunk 10]
```diff
@@ -0,0 +1,80 @@
...
+- Switch to Java 11 [#343](https://github.com/metafacture/metafacture-fix/issues/302)
```
Comment (blackwinter) [inline content/blog/metafacture-releases_core-6.0.0_fix-1.0.0_playground-1.0.0/index.md]:
```suggestion
- Switch to Java 11 [#343](https://github.com/metafacture/metafacture-fix/issues/343)
```

File: content/blog/metafacture-releases_core-6.0.0_fix-1.0.0_playground-1.0.0/index.md [hunk 11]
```diff
@@ -0,0 +1,80 @@
...
+- Add links to playground for several Fixes [#339](https://github.com/metafacture/metafacture-fix/issues/302)
```
Comment (blackwinter) [inline content/blog/metafacture-releases_core-6.0.0_fix-1.0.0_playground-1.0.0/index.md]:
```suggestion
- Add links to playground for several Fixes [#339](https://github.com/metafacture/metafacture-fix/issues/339)
```

File: content/blog/metafacture-releases_core-6.0.0_fix-1.0.0_playground-1.0.0/index.md [hunk 12]
```diff
@@ -0,0 +1,80 @@
...
+
+### Bug fixes
+
+- Close resources [#340](https://github.com/metafacture/metafacture-fix/issues/340)
+
+### Others
+
+- Update metafacture-core dependency to its latest release (6.0.0)
+
+## matafacture-playground
```
Comment (blackwinter) [inline content/blog/metafacture-releases_core-6.0.0_fix-1.0.0_playground-1.0.0/index.md]:
```suggestion
## metafacture-playground
```

File: content/blog/metafacture-releases_core-6.0.0_fix-1.0.0_playground-1.0.0/index.md [hunk 13]
```diff
@@ -0,0 +1,80 @@
...
+
+### New Features
+
+- Clear message and result when process [#126](https://github.com/metafacture/metafacture-playground/issues/126)
```
Comment (blackwinter) [inline content/blog/metafacture-releases_core-6.0.0_fix-1.0.0_playground-1.0.0/index.md]:
```suggestion
- Clear message and result when processing [#126](https://github.com/metafacture/metafacture-playground/issues/126)
```

**Review by blackwinter — CHANGES_REQUESTED** [review]
I don't get all the monospace formatting, but it seems to be established practice in these blog posts.

**Comment by acka47** [comment]
I think I will better take a look at it when @blackwinter 's suggestions are committed, @dr0i .

**Comment by dr0i** [comment]
Thx for the many fixes, especially fixing the links !

**Comment by dr0i** [comment]
@acka47 you can have a look now

**Review by acka47 — APPROVED** [review]
+1 I have removed the reference to the JSON-LD playground as there are a lot of plyaygrounds around and the JSON-LD one isn't even one of the best example, I think.

**Comment by dr0i** [comment]
Deployed, see https://blog.metafacture.org/. Closed.

**Pull request merged**